inspect_builder_test.go

```diff
@@ -76,17 +77,17 @@ func testInspectBuilder(t *testing.T, when spec.G, it spec.S) {
 
 				when("only deprecated lifecycle apis are present", func() {
 					it.Before(func() {
-						h.AssertNil(t, builderImage.SetLabel(
+						assert.Nil(builderImage.SetLabel(
```

Comment:
Should we maybe alias `Nil` with `Succeeds` for this use case? I'm not sure it reads especially well why we want this to be `nil`.